subsys/event_manager/event_manager.c

```diff
@@ -175,8 +175,8 @@ static void event_processor_fn(struct k_work *work)
 
 		bool consumed = false;
 
-		for (size_t prio = SUBS_PRIO_MIN;
-		     (prio <= SUBS_PRIO_MAX) && !consumed;
+		for (size_t prio = EVT_MGR_SUBS_PRIO_MIN;
```

Comment:
there is something wrong... these thing should be removed already. hasn't it been merged yet? we will need to rebase